templates/accounts/account/email.html

```diff
@@ -1,7 +1,6 @@
 {% extends "account/base.html" %}
 
 {% load i18n %}
-{% load url from future %}
```

Comment:
I think all of the templates/accounts/ files were copied in from a django app. Have you checked to see that there aren't other changes to these templates from that app? You may need to do some digging.